fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -593,12 +593,19 @@
     public static int max_running_txn_num_per_db = 100;
 
     /**
-     * The load task executor pool size. This pool size limits the max running load tasks.
-     * Currently, it only limits the load task of broker load, pending and loading phases.
+     * The pending_load task executor pool size. This pool size limits the max running pending_load tasks.
+     * Currently, it only limits the pending_load task of broker load and spark load.
      * It should be less than 'max_running_txn_num_per_db'
      */
     @ConfField(mutable = false, masterOnly = true)
-    public static int async_load_task_pool_size = 10;
+    public static int async_pending_load_task_pool_size = 5;
```

Comment:
Better to keep the config name. Or made some compatibility.
Some other users may modify this config to enhance their load ability.
If the name changed, their config will be disable and fall back to default value, this will cause problem

Reply:
And the `desired_max_waiting_jobs` is useless?

Reply:
> And the `desired_max_waiting_jobs` is useless?

desired_max_waiting_jobs is used to limit blocking queue size